scene/animation/animation_mixer.cpp

```diff
@@ -629,12 +629,13 @@ bool AnimationMixer::_update_caches() {
 			if (!track) {
 				Ref<Resource> resource;
 				Vector<StringName> leftover_path;
-				Node *child = parent->get_node_and_resource(path, resource, leftover_path);
+				Node *child = parent->get_node_or_null(path);
 
 				if (!child) {
-					ERR_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'.");
+					WARN_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'.");
 					continue;
 				}
+				parent->get_node_and_resource(path, resource, leftover_path);
```

Comment:
If the actual object for left_overpath is to be retrieved here, I think a check for the object is needed after that.